scripts/ci/update-github

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+set -ex
+
+TAG_NAME=$(git tag -l --contains HEAD)
+
+rm -rf dcos-nightly
+
+# Setup Git
+git config --global user.email "$GIT_USER@users.noreply.github.com"
+git config --global user.name "MesosphereCI Robot"
+git config credential.helper 'cache --timeout=300'
+git clone "https://$GIT_USER:$GIT_PASSWORD@github.com/mesosphere/dcos" dcos-nightly
+ls -l
+
+cd dcos-nightly || exit 1
+
+if [[ $TAG_NAME ]]
+then
+  git checkout -- .
+  git remote add upstream "https://$GIT_USER:$GIT_PASSWORD@github.com/dcos/dcos"
+  git checkout master
+  git checkout -b dcos-ui/$TAG_NAME
+
+  cp -fr ../buildinfo.json packages/dcos-ui/buildinfo.json
+
+  git add "packages/dcos-ui/buildinfo.json"
+  git commit -m "chore(dcos-ui): update package"
```

Comment:
(Optional) Can we add a commit body which states where the change is coming from and that it is automated?

Reply:
@bstavroulakis thoughts on this?

Reply:
Actually, I don't think need to do this. The commit author is `mesosphere-ci`, so we already know it's not a person committing, it's something automated.

<img width="438" alt="screen shot 2018-03-13 at 2 03 41 pm" src="https://user-images.githubusercontent.com/1527504/37369786-5d947714-26c7-11e8-9d03-e65eaae233eb.png">

Reply:
I just want to prevent a situation where someone asks "Who the hell is comitting this?" and it is not clear where this originates from. But I am not super strong with this opinion, so yeah nevermind